.travis.yml

```diff
@@ -1,11 +1,9 @@
 language: julia
 julia:
-  - 0.4
+  - 1.5.0
```

Comment:
just curious, does this work on earlier versions (like 1.2) or uses it an 1.5 feature?

Reply:
Good question, I’m not sure, I should find a way to test that . . . I imagine it’s at least compatible with a few previous versions, but I don’t know which ones exactly.

Reply:
@bfredl I looked at [Flux's](https://github.com/FluxML/Flux.jl/blob/cd811821a0ea150a72f8782b5ddf8b9bb5d1ef69/.travis.yml#L8) and [Gadfly's](https://github.com/GiovineItalia/Gadfly.jl/blob/ff151918958fb6d975d8e3af368d7eceecc877db/.travis.yml#L4) `.travis.yml`'s just to see what versions they were compatible with, and they both appear to be compatible with 1.0, so we could probably do the tests on `1` as well and see if that works. If not we can adjust them later.